openapi.yaml

```diff
@@ -14,6 +14,8 @@ tags:
     description: Secured Admin-only calls
   - name: researchers
     description: Operations available to regular researchers
+  - name: guests
```

Comment:
Are we differentiating internal guest or external guest?

Reply:
I didn't because the APIs we added are accessible to both. I guess I should probably add it since that's how it is.

Reply:
That makes sense. I'm good either way.